test/config-test.js

```diff
@@ -0,0 +1,11 @@
+import assert from 'assert'
+
+import {host} from './../src/config.js'
+assert.deepStrictEqual(host, 'localhost')
```

Comment:
Please use the typical Mocha test structure (with `describe` and `it`), and use the Chai module for assertions.